phpBB/includes/ucp/ucp_pm_compose.php

```diff
@@ -687,6 +687,27 @@ function compose_pm($id, $mode, $action, $user_folders = array())
 	$flash_status	= ($config['auth_flash_pm'] && $auth->acl_get('u_pm_flash')) ? true : false;
 	$url_status		= ($config['allow_post_links']) ? true : false;
 
+	/**
+	 * Event to override private message BBCode status indications
+	 *
+	 * @event core.ucp_pm_compose_modify_bbcode_status_indications
```

Comment:
I'd remove the `_indications` word because `status` says it all.

Applies to both files.